### PR TITLE
Drafts

### PR DESCRIPTION
**Pull request by SvenChmie**
Pull request of current status to allow commenting.

### CODE REVIEW

File: Guide_1_Send_Payment.md [hunk 1]
```diff
@@ -1,31 +1,112 @@
...
 ### Create the Payment Resource
 
+```python
+import math, random, requests
+
+### Replace these variables with your own data! ###
+auth_token = 'A VALID BEARER TOKEN HERE'
+organisation_id = 'YOUR ORGANISATION ID HERE'
+bank_id = 'YOUR UK SORTCODE HERE'
+bank_id_code = 'YOUR BIC HERE'
```
Comment (stevecookform3) [inline Guide_1_Send_Payment.md]: bank_id_code = GBDSC (thats the code that means the bank_id is a UK domestic sortcode). Might be worth some explanation of this during the introduction.
The bic isnt used during this tutorial (it is used for inbound payments though)

File: Guide_1_Send_Payment.md [hunk 2]
```diff
@@ -1,31 +1,112 @@
...
+```python
+import requests
 
-_(Note: Should we elaborate on how to do that?)_
+### Replace these variables with your own data! ###
+client_id = 'YOUR CLIENT ID HERE'
+client_secret = 'YOUR CLIENT SECRET HERE'
+
+auth_payload = "grant_type=client_credentials"
+auth_url = 'https://api.tabla.env.form3.tech/v1/oauth2/token'
```
Comment (stevecookform3) [inline Guide_1_Send_Payment.md]: we should change this to 'api.test.form3.tech'

File: Guide_1_Send_Payment.md (hunk 1 above)
Comment (SvenChmie) [reply Guide_1_Send_Payment.md]: I removed the variable and just use GBDSC in the payment attribute tree. Also added a sentence about it in the guide. What do you think?

File: Guide_1_Send_Payment.md (hunk 2 above)
Comment (SvenChmie) [reply Guide_1_Send_Payment.md]: Changed the URL in all snippets and code files.

File: Guide_1_Send_Payment.md [hunk 3]
```diff
@@ -1,31 +1,111 @@
...
-- Track the payment progress manually
-- Track the payment progress automatically using subscriptions
+Each step is illustrated with Python code snippets, so you can execute each step as you read along. The snippets are ready-to-run programs and work in Python's interactive console.
 
 ## Prerequisites:
 Before you start, make sure you have the following things ready to go:
-- API credentials. Contact your Form3 contact to obtain your credentials. _(Note: Better name for this role? Who do they have to contact?)_
-- Register your bank ID and BIC with Form3
-- Sign up for Form3's scheme simulator. This is a sandbox environment that you can use to simulate transactions in order to test your application. _(Note: in the API this is called "payment simulator". Which term should we use?)_
+- API credentials. Contact Form3 to obtain your credentials.
```
Comment (stevecookform3) [inline Guide_1_Send_Payment.md]: We should add the "Organisation" here - the unique ID for your account. Also will come from Form3.

File: Guide_1_Send_Payment.md [hunk 4]
```diff
@@ -1,31 +1,111 @@
...
+### Replace these variables with your own data! ###
+auth_token = 'A VALID BEARER TOKEN HERE'
+organisation_id = 'YOUR ORGANISATION ID HERE'
+bank_id = 'YOUR UK SORTCODE HERE'
+
+# Generate IDs
+payment_id = uuid.uuid4()
+print("Payment ID: %s" % payment_id)
+
+scheme_transaction_id = str(int(math.floor((1 + random.random()) * 100000000000000000)))
```
Comment (stevecookform3) [inline Guide_1_Send_Payment.md]: I think we now generate a scheme_transaction_id, so can you try without it and see if it still works? If so, you can remove that field.

File: Guide_1_Send_Payment.md [hunk 5]
```diff
@@ -37,11 +117,12 @@ The key parameters that you need to provide are:
...
+To identify the sending and receiving parties of the payment, you need to provide an `account_name`, the `account_number`, the `account_number_code`, as well as the `bank_id` and `bank_id_code` for the bank the account is registered with. 
+
+In this example, use your UK sortcode for the sending party's `bank_id`. 
+
+The `bank_id_code ` attribute denotes the type of the `bank_id`. Since only domestic UK accounts are used in this example, the `bank_id_code` is preset as `GBDSC`.
 
-```json
-Code for payment resource creation here!
-```
 Upon success, the call returns with a payment ID that you'll need below to send the payment.
```
Comment (stevecookform3) [inline Guide_1_Send_Payment.md]: Actually you already specify the payment ID before, so you dont need this here.

Comment (SvenChmie) [reply Guide_1_Send_Payment.md]: True, good point! I'll remove that.

File: Guide_1_Send_Payment.md (hunk 4 above)
Comment (SvenChmie) [reply Guide_1_Send_Payment.md]: I tested it, works without providing a scheme_transaction_id. I removed all mention of it in the guide and code.

File: Guide_1_Send_Payment.md (hunk 3 above)
Comment (SvenChmie) [reply Guide_1_Send_Payment.md]: Fixed!

**Pull request merged**